src/Perpetuum/Services/Weather/WeatherService.cs

```diff
@@ -18,27 +18,27 @@ static WeatherService()
             //the full cycle
             for (var i = 0; i < _weatherLookUp.Length; i++)
             {
-                _weatherLookUp[i] = (byte)i.Min(255);
+                _weatherLookUp[i] = (byte)i.Min(254);
```

Comment:
Found a little buggy bug, did we?

Reply:
Is there a reason we don't want to go to 255? 255 is a valid value for `Byte`

Reply:
Oh right, yes!  The client renders the most intense fog at 255 but it bugs out.
It's weird, but if you pull this down and up the limit and run the set command for 255, 255, once it hits the top value it will be very foggy but then suddenly disappear.  I think the client doesn't do well with the weather value at max in this case..